src/sparsevector.jl

```diff
@@ -114,6 +117,9 @@ function nnz(x::SparseColumnView)
     return length(nzrange(parent(x), colidx))
 end
 nnz(x::SparseVectorView) = nnz(x.parent)
+function nnz(x::SparseVectorPartialView)
```

Comment:
some bikeshedding :P but write this as a one liner IMHO

Reply:
done indeed